stagemonitor-requestmonitor/src/main/java/org/stagemonitor/requestmonitor/MonitoredRequest.java

```diff
@@ -34,6 +36,8 @@
 	 */
 	Object execute() throws Exception;
```

Comment:
Is it necessary to differ between execute and executeAsync or could we just refactor execute to return a Future which can be either a `org.stagemonitor.core.util.CompletedFuture` or a real future?